Backend/Backend/Models/Playlist.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Backend.Infrastructure;
+
+namespace Backend.Models
+{
+    public class Playlist : BaseEntity
+    {
+        [Key]
+        public Guid Id { get; set; }
+        
+        public Guid UserId { get; set; }
+
+        public User? User => _context?.Users?.Where(u => u.Id == UserId).FirstOrDefault();
+        
+        public string Name { get; set; }
+
+        public List<PlaylistElement>? Elements => _context?.PlaylistElements.Where(e => e.PlayerlistId == Id).ToList();
```

Comment:
We should do all of our DB requests async.

Reply:
This is a field not a method that's the issue